test/unit/api-config-test.js

```diff
@@ -133,6 +133,18 @@ describe('API Config', function() {
         });
     });
 
+    describe('playbackRates', function() {
+
+        it('should round custom playback rates to nearest hundredth', function() {
+            let rates = [ 0.5, 0.6512, 1 ];
+            rates = rates.filter(rate => rate >= 0.25 && rate <= 4)
+                .map(rate => Math.round(rate * 100) / 100);
+            expect(rates[0]).to.equal(0.50);
+            expect(rates[1]).to.equal(0.65);
+            expect(rates[2]).to.equal(1.00);
```

Comment:
Add an assertion to show it's actually rounding. `0.6512 => 0.65` could be flooring the value. `0.999 => 1` would show it rounds. `0.667 => 0.67` and `0.664 => 0.66` would show it rounds to the nearest hundredth.